components/pd_client/src/client.rs

```diff
@@ -282,6 +283,62 @@ impl fmt::Debug for RpcClient {
 const LEADER_CHANGE_RETRY: usize = 10;
 
 impl PdClient for RpcClient {
+    fn load_global_config(&self, list: Vec<String>) -> Result<HashMap<String, String>> {
+        use kvproto::pdpb::LoadGlobalConfigRequest;
+        let mut req = LoadGlobalConfigRequest::new();
+        for elem in list {
+            req.names.push(elem);
```

Comment:
maybe we can directly use `append` to add the whole vec

Reply:
doesn't exist append method

Reply:
The whole loop can be replaced with `req.set_names(list.into());`